src/docbuilder/rustwide_builder.rs

```diff
@@ -1044,6 +1069,37 @@ mod tests {
         });
     }
 
+    #[test]
+    #[ignore]
+    fn test_cdn_invalidation() {
+        wrapper(|env| {
+            env.override_config(|cfg| {
+                cfg.cdn_backend = CdnKind::Dummy;
+                cfg.cloudfront_distribution_id_web = Some("distribution_id".into());
+            });
+
+            let mut builder = RustwideBuilder::init(env).unwrap();
+            assert!(builder.build_package(
+                DUMMY_CRATE_NAME,
+                DUMMY_CRATE_VERSION,
+                PackageKind::CratesIo
+            )?);
+
+            assert!(matches!(*env.cdn(), CdnBackend::Dummy(_)));
+            if let CdnBackend::Dummy(ref invalidation_requests) = *env.cdn() {
+                let ir = invalidation_requests.lock().unwrap();
+                assert_eq!(ir.len(), 2);
+                assert_eq!(ir[0], ("distribution_id".into(), "/empty-library*".into()));
+                assert_eq!(
+                    ir[1],
+                    ("distribution_id".into(), "/crate/empty-library*".into())
+                );
```

Comment:
You can just assert the whole list at once:

```suggestion
                assert_eq!(ir, [
                    ("distribution_id".into(), "/empty-library*".into()),
                    ("distribution_id".into(), "/crate/empty-library*".into()),
                ]);
```